net.sf.eclipsecs.core/src/net/sf/eclipsecs/core/config/meta/MetadataFactory.java

```diff
@@ -100,6 +134,36 @@ private MetadataFactory() {
     refresh();
   }
 
+  /*
+   * Fetch third party checks extension metadata from YML files.
+   */
+  private static void getThirdPartyModuleExtensionMetadata() {
+    Set<String> eclipseMetaDataFiles = new HashSet<>();
+
+    for (String packageName : sPackageNameSet) {
+      if (packageName.matches(".*\\.checkstyle.*")) {
```

Comment:
Please explain this 
if you filterout standard Checkstyle packge - please use full name of package.
if not ... we can not limit user to keep "checkstyle" as art of their package name.

Reply:
My bad, I have removed it.